onos-classic/templates/statefulset.yaml

```diff
@@ -71,8 +71,8 @@ spec:
         env:
           - name: JAVA_OPTS
             value: -Xmx{{ .Values.heap }}
-          - name: ONOS_APPS
```

Comment:
don't we still want the ability to set `ONOS_APPS` as part of the chart install?

Reply:
Hi David,
The liveliness apps list is set through "helm --set".In case the "livelinessCheckApps" in values.yml is empty and ONOS_APPS env is set through helm, we consider the ONOS_APPS.

The below code snippet, considers the ONOS_APPS in case the "LIVELINESS_CHECK_APPS" values are empty.
---------------------------
onos-classic/templatess/configmap-live-probe.yaml : line no: 17
apps=$LIVELINESS_CHECK_APPS
        if [[ -z "$apps" ]]; then
          apps=$ONOS_APPS
        fi

Reply:
So there is no way to override or set the ONOS_APPS via a `--set` option to helm install? If not it seems like you are pushing people toward either (a) having a post install k8s jobs or other mechanism to "setup" (activate apps) after is is started or (b) have people build custom images. 

It seems to me that it would be useful to be able to set onos apps and an initial netcfg when you `helm install`. the netcfg (as well as settings) could be established as a user settable configmap for example.

Reply:
Hi David,
Sorry for the confusion caused. This change is to verify the liveliness check for the app fetched from values.livelinessCheckApps variables. 
We don't have the ONOS_APPS variable set anywhere in the charts. Hence we removed code snippet from the file "onos-classic/templates/configmap-live-probe.yaml ".

The "values.livelinessCheckApps" values can be set dynamically from "helm install --set".